relayer/src/setup/index.ts

```diff
@@ -1,8 +1,81 @@
-import { createTestAccounts } from "@lightprotocol/zk.js";
-import { getAnchorProvider } from "../utils/provider";
+import {
+  createTestAccounts,
+  initLookUpTable,
+  useWallet,
+  airdropSol,
+} from "@lightprotocol/zk.js";
+import {
+  getAnchorProvider,
+  getKeyPairFromEnv,
+  getRelayer,
+} from "../utils/provider";
+import { PublicKey } from "@solana/web3.js";
+import { readFileSync, writeFileSync } from "fs";
+import { AnchorProvider, BN } from "@coral-xyz/anchor";
 
 export const testSetup = async () => {
-  const providerAnchor = await getAnchorProvider();
-  // TODO: use updated -- buildscript -> add relayer tests
-  await createTestAccounts(providerAnchor.connection);
+  const anchorProvider = await getAnchorProvider();
```

Comment:
Are we using the relayer test setup anywhere?
(if not let's remove)

Reply:
yes, renamed to relayerSetup (e.g. inits relayerRecipient when relayer starts) this was previously one source of error since for example the CLI tests start the relayer but don't execute Pre-hooks found in other tests